app_dart/lib/src/model/firestore/task.dart

```diff
@@ -254,6 +271,27 @@ class Task extends Document {
     }
   }
 
+  String _setStatusFromLuciStatusV2(bbv2.Build build) {
+    // Updates can come out of order. Ensure completed statuses are kept.
+    if (_isStatusCompleted()) {
+      return status;
+    }
+
+    if (build.status == bbv2.Status.STARTED) {
+      return setStatus(statusInProgress);
+    } else if (build.status == bbv2.Status.SUCCESS) {
```

Comment:
Is there still the `build.result` field? or are they merged?

Reply:
No the Result field was deprecated.

Reply:
Gotcha. That makes the logic easier.